src/psyclone/domain/lfric/lfric_kern.py

```diff
@@ -609,14 +609,14 @@ def gen_stub(self):
         # Add all the declarations
```

Comment:
Strangely my pylint crashes when processing this file - but that's obvious not your problem :)

Reply:
Here's what I get:

```
src/psyclone/domain/lfric/lfric_kern.py:267:13: W0511: TODO: #11 add a warning here when logging is added (fixme)
src/psyclone/domain/lfric/lfric_kern.py:309:17: W0511: TODO #719 we don't yet have a symbol table to prevent (fixme)
src/psyclone/domain/lfric/lfric_kern.py:592:9: W0511: TODO #925 Add support for 'domain' kernels (fixme)
src/psyclone/domain/lfric/lfric_kern.py:746:13: W0511: TODO #2271 could potentially avoid the need for (fixme)
src/psyclone/domain/lfric/lfric_kern.py:761:13: W0511: TODO #928: We don't validate the arguments yet because the (fixme)
src/psyclone/domain/lfric/lfric_kern.py:782:9: W0511: TODO #935 - replace the PSyIR argument data symbols with LFRic data (fixme)
src/psyclone/domain/lfric/lfric_kern.py:877:13: W0511: TODO #1941: it would be better if the LFRic constants_mod.f90 (fixme)
src/psyclone/domain/lfric/lfric_kern.py:60:0: W0223: Method 'local_vars' is abstract in class 'Kern' but is not overridden (abstract-method)
src/psyclone/domain/lfric/lfric_kern.py:578:4: R0914: Too many local variables (19/15) (too-many-locals)
src/psyclone/domain/lfric/lfric_kern.py:841:4: R0914: Too many local variables (16/15) (too-many-locals)
src/psyclone/domain/lfric/lfric_kern.py:841:4: R0912: Too many branches (17/12) (too-many-branches)
src/psyclone/domain/lfric/lfric_kern.py:60:0: R0904: Too many public methods (22/20) (too-many-public-methods)
```

Reply:
Great. No idea why my pylint is broken. Updating didn't fix it. But it looks all good - thanks!